webui/react/src/components/CreateExperimentModal.tsx

```diff
@@ -14,7 +14,7 @@ export enum CreateExperimentType {
   ContinueTrial = 'Continue Trial',
 }
 
-interface Props extends ModalFuncProps {
+interface Props extends Omit<ModalFuncProps, 'type'> {
```

Comment:
`ModalFuncProps` has a `type` prop defined as well, new TypeScript complains about this.

Reply:
me too lol